samples/CommunityToolkit.Maui.Sample/Pages/Views/AvatarView/AvatarViewBindablePropertiesPage.xaml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.Views.AvatarViewBindablePropertiesPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+    xmlns:models="clr-namespace:CommunityToolkit.Maui.Sample.Models"
+    xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
+    xmlns:vm="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Views.AvatarView"
+    Title="AvatarView Bindable"
+    x:DataType="vm:AvatarViewBindablePropertiesViewModel"
+    x:TypeArguments="vm:AvatarViewBindablePropertiesViewModel">
+    <VerticalStackLayout Spacing="12">
+        <VerticalStackLayout.Resources>
+            <ResourceDictionary>
+                <Style x:Key="Description" TargetType="Label">
+                    <Setter Property="VerticalTextAlignment" Value="Start" />
+                    <Setter Property="HorizontalTextAlignment" Value="Start" />
+                    <Setter Property="LineBreakMode" Value="WordWrap" />
+                    <Setter Property="Margin" Value="4,0" />
+                </Style>
+                <Style x:Key="Divider" TargetType="BoxView">
```

Comment:
From docs: Although BoxView can mimic simple graphics, a better alternative is to use .NET MAUI Shapes or .NET MAUI Graphics.

Reply:
This was simply copied directly from `ButtonPopup.xaml` and the other pages.  It's just used as a 'divider'.  Happy to change to something else you suggest.

Reply:
If you can, let's replace it with a `Line`: https://docs.microsoft.com/dotnet/maui/user-interface/controls/shapes/line?WT.mc_id=dotnet-0000-bramin

Reply:
In `AvatarView*Page` all `BoxView` has been replaced with `Line`.